synapse/handlers/e2e_keys.py

```diff
@@ -174,8 +182,8 @@ def do_remote_query(destination):
             """This is called when we are querying the device list of a user on
             a remote homeserver and their device list is not in the device list
             cache. If we share a room with this user and we're not querying for
-            specific user we will update the cache
-            with their device list."""
+            specific user we will update the cache with their device list.
```

Comment:
I'm assuming you've grokked what this sentence was trying to say so:

what does "we're not querying for specific user" [sic] mean?

surely it's meaningless to query for keys that don't belong to a specific user?

Reply:
This is a very good question, but I'm afraid I just fixed this comment as it had a broken line break.

Actually looking at it though... I agree that the wording doesn't really make any sense and is redundant. I think all we need to say is that we're only going to update their device list if we share a room with them.